Cleanup Compiler Warnings:

Saving output here as I've been meaning to clean them up but they keep getting pushed down the priority list.

```
--- stderr: kortex2_driver                                                                                                                                                                    
/home/alex/ros/h/c3p0/src/ros2_kortex/kortex2_driver/src/kortex_math_util.cpp: In static member function ‘static int KortexMathUtil::getNumberOfTurns(double)’:
/home/alex/ros/h/c3p0/src/ros2_kortex/kortex2_driver/src/kortex_math_util.cpp:13:45: warning: parameter ‘rad_not_wrapped’ set but not used [-Wunused-but-set-parameter]
   13 | int KortexMathUtil::getNumberOfTurns(double rad_not_wrapped)
      |                                      ~~~~~~~^~~~~~~~~~~~~~~
In file included from /home/alex/ros/h/c3p0/src/ros2_kortex/kortex2_driver/include/kortex2_driver/hardware_interface.hpp:46,
                 from /home/alex/ros/h/c3p0/src/ros2_kortex/kortex2_driver/src/hardware_interface.cpp:33:
/home/alex/ros/h/c3p0/install/kortex_api/include/BaseClientRpc.h: In member function ‘void Kinova::Api::Base::BaseClient::messageHeaderValidation(const Kinova::Api::Frame&)’:
/home/alex/ros/h/c3p0/install/kortex_api/include/BaseClientRpc.h:1262:45: warning: unused parameter ‘msgFrame’ [-Wunused-parameter]
 1262 |   void messageHeaderValidation(const Frame& msgFrame)
      |                                ~~~~~~~~~~~~~^~~~~~~~
In file included from /home/alex/ros/h/c3p0/src/ros2_kortex/kortex2_driver/include/kortex2_driver/hardware_interface.hpp:47,
                 from /home/alex/ros/h/c3p0/src/ros2_kortex/kortex2_driver/src/hardware_interface.cpp:33:
/home/alex/ros/h/c3p0/install/kortex_api/include/BaseCyclicClientRpc.h: In member function ‘void Kinova::Api::BaseCyclic::BaseCyclicClient::messageHeaderValidation(const Kinova::Api::Frame&)’:
/home/alex/ros/h/c3p0/install/kortex_api/include/BaseCyclicClientRpc.h:86:45: warning: unused parameter ‘msgFrame’ [-Wunused-parameter]
   86 |   void messageHeaderValidation(const Frame& msgFrame)
      |                                ~~~~~~~~~~~~~^~~~~~~~
In file included from /home/alex/ros/h/c3p0/install/kortex_api/include/SessionManager.h:13,
                 from /home/alex/ros/h/c3p0/src/ros2_kortex/kortex2_driver/include/kortex2_driver/hardware_interface.hpp:49,
                 from /home/alex/ros/h/c3p0/src/ros2_kortex/kortex2_driver/src/hardware_interface.cpp:33:
/home/alex/ros/h/c3p0/install/kortex_api/include/SessionClientRpc.h: In member function ‘void Kinova::Api::Session::SessionClient::messageHeaderValidation(const Kinova::Api::Frame&)’:
/home/alex/ros/h/c3p0/install/kortex_api/include/SessionClientRpc.h:81:45: warning: unused parameter ‘msgFrame’ [-Wunused-parameter]
   81 |   void messageHeaderValidation(const Frame& msgFrame)
      |                                ~~~~~~~~~~~~~^~~~~~~~
In file included from /home/alex/ros/h/c3p0/src/ros2_kortex/kortex2_driver/include/kortex2_driver/hardware_interface.hpp:51,
                 from /home/alex/ros/h/c3p0/src/ros2_kortex/kortex2_driver/src/hardware_interface.cpp:33:
/home/alex/ros/h/c3p0/install/kortex_api/include/TransportClientUdp.h: In member function ‘virtual char* Kinova::Api::TransportClientUdp::getTxBuffer(const uint32_t&)’:
/home/alex/ros/h/c3p0/install/kortex_api/include/TransportClientUdp.h:121:45: warning: unused parameter ‘allocation_size’ [-Wunused-parameter]
  121 |   virtual char* getTxBuffer(uint32_t const& allocation_size) override
      |                             ~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~
In file included from /home/alex/ros/h/c3p0/src/ros2_kortex/kortex2_driver/src/hardware_interface.cpp:33:
/home/alex/ros/h/c3p0/src/ros2_kortex/kortex2_driver/include/kortex2_driver/hardware_interface.hpp: In constructor ‘kortex2_driver::KortexMultiInterfaceHardware::KortexMultiInterfaceHardware()’:
/home/alex/ros/h/c3p0/src/ros2_kortex/kortex2_driver/include/kortex2_driver/hardware_interface.hpp:196:8: warning: ‘kortex2_driver::KortexMultiInterfaceHardware::use_internal_bus_gripper_comm_’ will be initialized after [-Wreorder]
  196 |   bool use_internal_bus_gripper_comm_;
      |        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/alex/ros/h/c3p0/src/ros2_kortex/kortex2_driver/include/kortex2_driver/hardware_interface.hpp:125:30: warning:   ‘Kinova::Api::Base::Twist* kortex2_driver::KortexMultiInterfaceHardware::k_api_twist_’ [-Wreorder]
  125 |   Kinova::Api::Base::Twist * k_api_twist_;
      |                              ^~~~~~~~~~~~
/home/alex/ros/h/c3p0/src/ros2_kortex/kortex2_driver/src/hardware_interface.cpp:46:1: warning:   when initialized here [-Wreorder]
   46 | KortexMultiInterfaceHardware::KortexMultiInterfaceHardware()
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/alex/ros/h/c3p0/src/ros2_kortex/kortex2_driver/src/hardware_interface.cpp: In member function ‘virtual hardware_interface::return_type kortex2_driver::KortexMultiInterfaceHardware::read(const rclcpp::Time&, const rclcpp::Duration&)’:
/home/alex/ros/h/c3p0/src/ros2_kortex/kortex2_driver/src/hardware_interface.cpp:741:24: warning: unused parameter ‘time’ [-Wunused-parameter]
  741 |   const rclcpp::Time & time, const rclcpp::Duration & period)
      |   ~~~~~~~~~~~~~~~~~~~~~^~~~
/home/alex/ros/h/c3p0/src/ros2_kortex/kortex2_driver/src/hardware_interface.cpp:741:55: warning: unused parameter ‘period’ [-Wunused-parameter]
  741 |   const rclcpp::Time & time, const rclcpp::Duration & period)
      |                              ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~
/home/alex/ros/h/c3p0/src/ros2_kortex/kortex2_driver/src/hardware_interface.cpp: In member function ‘virtual hardware_interface::return_type kortex2_driver::KortexMultiInterfaceHardware::write(const rclcpp::Time&, const rclcpp::Duration&)’:
/home/alex/ros/h/c3p0/src/ros2_kortex/kortex2_driver/src/hardware_interface.cpp:797:24: warning: unused parameter ‘time’ [-Wunused-parameter]
  797 |   const rclcpp::Time & time, const rclcpp::Duration & period)
      |   ~~~~~~~~~~~~~~~~~~~~~^~~~
/home/alex/ros/h/c3p0/src/ros2_kortex/kortex2_driver/src/hardware_interface.cpp:797:55: warning: unused parameter ‘period’ [-Wunused-parameter]
  797 |   const rclcpp::Time & time, const rclcpp::Duration & period)
      |                              ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~
---
```

--- a/kortex_driver/CMakeLists.txt
+++ b/kortex_driver/CMakeLists.txt
@@ -34,8 +34,8 @@ target_include_directories(
 )
 ament_target_dependencies(
   ${PROJECT_NAME}
+  SYSTEM kortex_api
   hardware_interface
-  kortex_api
   pluginlib
   rclcpp
 )

--- a/kortex_driver/src/hardware_interface.cpp
+++ b/kortex_driver/src/hardware_interface.cpp
@@ -52,8 +52,12 @@ KortexMultiInterfaceHardware::KortexMultiInterfaceHardware()
     &transport_udp_realtime_,
     [](k_api::KError err) { cout << "_________ callback error _________" << err.toString(); }},
   session_manager_real_time_{&router_udp_realtime_},
+  k_api_twist_(nullptr),
   base_{&router_tcp_},
   base_cyclic_{&router_udp_realtime_},
+  gripper_motor_command_(nullptr),
+  gripper_command_max_velocity_(100.0),
+  gripper_command_max_force_(100.0),
   servoing_mode_hw_(k_api::Base::ServoingModeInformation()),
   joint_based_controller_running_(false),
   twist_controller_running_(false),
@@ -69,11 +73,7 @@ KortexMultiInterfaceHardware::KortexMultiInterfaceHardware()
   start_fault_controller_(false),
   first_pass_(true),
   gripper_joint_name_(""),
-  gripper_command_max_velocity_(100.0),
-  gripper_command_max_force_(100.0),
-  use_internal_bus_gripper_comm_(false),
-  k_api_twist_(nullptr),
-  gripper_motor_command_(nullptr)
+  use_internal_bus_gripper_comm_(false)
 {
   RCLCPP_INFO(LOGGER, "Setting severity threshold to DEBUG");
   auto ret = rcutils_logging_set_logger_level(LOGGER.get_name(), RCUTILS_LOG_SEVERITY_DEBUG);
@@ -743,7 +743,7 @@ CallbackReturn KortexMultiInterfaceHardware::on_deactivate(
 }
 
 return_type KortexMultiInterfaceHardware::read(
-  const rclcpp::Time & time, const rclcpp::Duration & period)
+  const rclcpp::Time & /*time*/, const rclcpp::Duration & /*period*/)
 {
   if (first_pass_)
   {
@@ -799,7 +799,7 @@ void KortexMultiInterfaceHardware::readGripperPosition()
 }
 
 return_type KortexMultiInterfaceHardware::write(
-  const rclcpp::Time & time, const rclcpp::Duration & period)
+  const rclcpp::Time & /*time*/, const rclcpp::Duration & /*period*/)
 {
   if (block_write)
   {

